testsuite/openshift/__init__.py

```diff
@@ -16,22 +16,30 @@ class OpenShiftObject(APIObject, LifecycleObject):
 
     def __init__(self, dict_to_model=None, string_to_model=None, context=None):
         super().__init__(dict_to_model, string_to_model, context)
-        self.committed = False
+        self._committed = None
+
+    @property
+    def committed(self):
+        """Returns True, if the objects is already committed to the server"""
+        if self._committed is None:
+            exists, _ = self.exists()
```

Comment:
This can be a one-liner. 
I am thinking of renaming `_committed` to `_exists`? But it may be BS.

Reply:
how can this be a one liner?

Reply:
`self._committed, _ = self.exists()`

Reply:
Thanks, I did not think of that